2024-11-14 20:11:45,612 [INFO] TensorFlow version: 2.13.0
2024-11-14 20:11:45,614 [INFO] GPU available: []
2024-11-14 20:11:45,615 [ERROR] ¬°¬ê¬Ú¬Ò¬Ü¬Ñ ¬Ó ¬á¬â¬à¬è¬Ö¬ã¬ã¬Ö ¬Ü¬à¬ß¬Ó¬Ö¬â¬ä¬Ñ¬è¬Ú¬Ú: ¬®¬à¬Õ¬Ö¬Ý¬î ¬ß¬Ö ¬ß¬Ñ¬Û¬Õ¬Ö¬ß¬Ñ: best_model.keras
2024-11-14 20:11:45,615 [ERROR] Traceback (most recent call last):
  File "converter.py", line 29, in convert_to_tflite
    raise FileNotFoundError(f"¬®¬à¬Õ¬Ö¬Ý¬î ¬ß¬Ö ¬ß¬Ñ¬Û¬Õ¬Ö¬ß¬Ñ: {keras_model_path}")
FileNotFoundError: ¬®¬à¬Õ¬Ö¬Ý¬î ¬ß¬Ö ¬ß¬Ñ¬Û¬Õ¬Ö¬ß¬Ñ: best_model.keras

2024-11-14 20:11:45,615 [ERROR] ¬±¬â¬à¬Ô¬â¬Ñ¬Þ¬Þ¬Ñ ¬Ù¬Ñ¬Ó¬Ö¬â¬ê¬Ö¬ß¬Ñ ¬ã ¬à¬ê¬Ú¬Ò¬Ü¬Ñ¬Þ¬Ú
2024-11-14 20:12:10,588 [INFO] TensorFlow version: 2.13.0
2024-11-14 20:12:10,589 [INFO] GPU available: []
2024-11-14 20:12:10,589 [INFO] ¬¯¬Ñ¬é¬Ú¬ß¬Ñ¬Ö¬Þ ¬Ü¬à¬ß¬Ó¬Ö¬â¬ä¬Ñ¬è¬Ú¬ð ¬Þ¬à¬Õ¬Ö¬Ý¬Ú: best_model.keras
2024-11-14 20:12:10,589 [INFO] ¬²¬Ñ¬Ù¬Þ¬Ö¬â ¬Ú¬ã¬ç¬à¬Õ¬ß¬à¬Ô¬à ¬æ¬Ñ¬Û¬Ý¬Ñ: 295.22 MB
2024-11-14 20:12:10,589 [INFO] ¬©¬Ñ¬Ô¬â¬å¬Ø¬Ñ¬Ö¬Þ Keras ¬Þ¬à¬Õ¬Ö¬Ý¬î...
2024-11-14 20:12:11,806 [INFO] ¬®¬à¬Õ¬Ö¬Ý¬î ¬å¬ã¬á¬Ö¬ê¬ß¬à ¬Ù¬Ñ¬Ô¬â¬å¬Ø¬Ö¬ß¬Ñ
2024-11-14 20:12:11,806 [INFO] ¬£¬ç¬à¬Õ¬ß¬Ñ¬ñ ¬â¬Ñ¬Ù¬Þ¬Ö¬â¬ß¬à¬ã¬ä¬î ¬Þ¬à¬Õ¬Ö¬Ý¬Ú: (None, 224, 224, 3)
2024-11-14 20:12:11,806 [WARNING] ¬£¬ß¬Ú¬Þ¬Ñ¬ß¬Ú¬Ö: ¬Ó¬ç¬à¬Õ¬ß¬Ñ¬ñ ¬â¬Ñ¬Ù¬Þ¬Ö¬â¬ß¬à¬ã¬ä¬î ¬Þ¬à¬Õ¬Ö¬Ý¬Ú ¬ß¬Ö 299x299!
2024-11-14 20:12:11,807 [INFO] ¬ª¬ß¬Ú¬è¬Ú¬Ñ¬Ý¬Ú¬Ù¬Ñ¬è¬Ú¬ñ ¬Ü¬à¬ß¬Ó¬Ö¬â¬ä¬Ö¬â¬Ñ...
2024-11-14 20:12:11,807 [INFO] ¬¯¬Ñ¬ã¬ä¬â¬à¬Û¬Ü¬Ñ ¬á¬Ñ¬â¬Ñ¬Þ¬Ö¬ä¬â¬à¬Ó ¬Ü¬à¬ß¬Ó¬Ö¬â¬ä¬Ñ¬è¬Ú¬Ú...
2024-11-14 20:12:11,807 [ERROR] ¬°¬ê¬Ú¬Ò¬Ü¬Ñ ¬Ó ¬á¬â¬à¬è¬Ö¬ã¬ã¬Ö ¬Ü¬à¬ß¬Ó¬Ö¬â¬ä¬Ñ¬è¬Ú¬Ú: module 'tensorflow._api.v2.lite' has no attribute 'Spec'
2024-11-14 20:12:11,807 [ERROR] Traceback (most recent call last):
  File "converter.py", line 63, in convert_to_tflite
    converter.optimization_default_target_spec = tf.lite.Spec()
AttributeError: module 'tensorflow._api.v2.lite' has no attribute 'Spec'

2024-11-14 20:12:11,829 [ERROR] ¬±¬â¬à¬Ô¬â¬Ñ¬Þ¬Þ¬Ñ ¬Ù¬Ñ¬Ó¬Ö¬â¬ê¬Ö¬ß¬Ñ ¬ã ¬à¬ê¬Ú¬Ò¬Ü¬Ñ¬Þ¬Ú
